typescript/rest-express/src/index.ts

```diff
@@ -48,6 +84,27 @@ app.delete(`/post/:id`, async (req, res) => {
   res.json(post)
 })
 
+app.get('/users', async (req, res) => {
+  const users = await prisma.user.findMany()
+  res.json(users)
+})
+
+app.get('/user/:id/drafts', async (req, res) => {
+  const { id } = req.params
+  const { email } = req.body
+
+  const drafts = await prisma.user.findUnique({
+    where: {
+      id: Number(id) || undefined,
+      email: email || undefined
```

Comment:
can you double check whether the `|| undefined` is actually needed here? I believe this was specific to Nexus so we _should_ be able to omit it in non-Nexus project. if it doesn't produce any compile-time or runtime errors, we can leave it out.

Reply:
This isn't needed. Thank you for spotting it. 🙂